poll/public/css/poll.css

```diff
@@ -146,7 +146,7 @@ li.poll-result .poll-image {
     border-top: 0;
 }
 
-.survey-table tr:last-child td {
+.survey-table > tbody > tr:last-child td {
```

Comment:
Are the `>` necessary here? This makes for very rigid structure. Can we just use normal nesting:

`.survey-table tr:last-child td`